.travis.yml

```diff
@@ -9,25 +9,36 @@ env:
 
 jobs:
   include:
-    - stage: "OpenShift and Kubernetes CI"
+    - stage: "Format"
+      script:
+        - ./scripts/travis.sh format
+    - stage: "Vet"
+      script:
+        - ./scripts/travis.sh vet
+    - stage: "Lint"
+      script:
+        - ./scripts/travis.sh lint
+    - stage: "Build"
+      script:
+        - ./scripts/travis.sh build
+    - stage: "Test"
+      script:
+        - ./scripts/travis.sh build
+
+    - stage: "OpenShift CI"
```

Comment:
I think this comment is outside the scope of this PR, but I was thinking it would be nice if these "OpenShift CI" and "Kubernetes CI" were separate tests belonging to the same stage "Functional Testing".